src/statement.c

```diff
@@ -3206,6 +3206,40 @@ Statement *PragmaStatement::semantic(Scope *sc)
             return this;
         }
     }
+    else if (ident == Id::Pinline)
+    {
+        PINLINE inlining = PINLINEdefault;
+        if (!args || args->dim == 0)
+            inlining = PINLINEdefault;
+        else if (!args || args->dim != 1)
+        {
+            error("boolean expression expected for pragma(inline)");
+            goto Lerror;
+        }
+        else
+        {
+            Expression *e = (*args)[0];
```

Comment:
That's right.
1. it won't work, because of newScope()
2. can't think of a use case for it anyway